tests/installation/partitioning_raid.pm

```diff
@@ -211,51 +210,58 @@ sub run {
     }
 
     for (qw(vda vdb vdc vdd)) {
-        assert_screen "partitioning_raid-disk_$_-selected";
+        my $timeout = 2;
+        my $counter = 50;
```

Comment:
Do we really need 50 steps to navigate to the disk? As we go in one direction only, probably we don't need that much, as if needle doesn't match, we will do a lot of unnecessary steps with 2 seconds timeout.

Reply:
I've tried different Settings here and those were the most reliable.
Might be that we can decrease it in the future (with a better qemu version) but for now I would keep it